.github/workflows/report_quiz/common/utils.sh

```diff
@@ -0,0 +1,5 @@
+# To escapes some stupid quotes
```

Comment:
Am wondering if this is really needed in a separate file...

Reply:
This command is used in many files.
With future features, other commands will join it.

Reply:
Don't code for the future.
rn I see no reason to have a separate file for a single bash command

Reply:
Should repeat the function? 🤔

Reply:
Before trying to DRY something, I think you should ask yourself how often the module is going to change over time
it's used 2 times only right ?

Also... maybe am the only one thinking like that but I had too much bad surprise trying to DRY in SHELL programming... it's not a big deal, so do as you wish 😎